### DISCUSSION TITLE
Carla VST UI Wrapper Info

### DISCUSSION THREAD

**Discussion by MusoCity** [falkTX/Carla#1525] [Q&A] [answered]
I'm trying to get the makers of Band in a Box to create a vst to send VstTimeInfo to the main bbw.exe to sync playback to the DAW. I see Carla uses this method like some other standalone apps use to create a vst version.
Is there any info about the wrapper I can send to the makers of Biab to help them implement this ?
Any info would be much appreciated.

**Comment by falkTX** [comment, marked as answer]
This is just the normal way for vst plugins to implement things, there is nothing specific to carla about it.

**Reply by MusoCity** [reply]
Is there any information I can send to the to give them an idea how to implement it ?

**Reply by falkTX** [reply]
You should tell them that making new VST2 plugins is no longer legally possible.
The only workaround we have to far is to not use the VST2 SDK and instead use alternative GPL-licensed headers, but that makes it so that products based on it need to be GPL-licensed as well (read: open-source)

The VST2 API is quite simple and straightforward, you do not need to send any information. Any developer can just read VstTimeInfo and how to request such info from the host. It is literally just 1 function to get the information.

**Reply by MusoCity** [reply]
Thanks.

**Reply by MusoCity** [reply]
EDIT: worked that out

**Reply by MusoCity** [reply]
Also in Plugin mode should there be an "Always On Top" pin ? I have to use AutoHotkey Winset, Alwaysontop, , A
